### DISCUSSION TITLE
Roles & Permissions

### DISCUSSION THREAD

**Discussion by dgrigorov** [Laravel-Backpack/community-forum#821] [Q&A (Help)] [answered]
Hello guys and ladies, I really love your product but I couldn't find anything about roles and permissions. Do you have that feature in the project? Super Admin, Admin and Normal customer for example?

**Comment by karandatwani92** [comment, marked as answer]
Hey @dgrigorov 

Thanks for your kind words. **Yes**, Backpack has it as an add-on([PermissionManager](https://github.com/Laravel-Backpack/PermissionManager)). This will let you have roles and permission features ready.

You can find more add-ons here.
https://backpackforlaravel.com/docs/6.x/add-ons-official
https://backpackforlaravel.com/addons

Cheers!
